### DISCUSSION TITLE
Displaying images as-is

### DISCUSSION THREAD

**Discussion by Aman-Aalam** [gethinode/hinode#811] [Help] [answered]
Hi Team,

Thank you for your previous responses, and I am sorry if this has been answered already.
I tried to search but didn't find a similar existing issue.

How do I display images as-is in my post content?
I need to display diagrams, and most of the ratios I provide are cutting the edges off that diagram.
If I don't provide any ratio, some default ratio gets applied.

Is there a way to do `width:100%` with the images without writing the HTML `img` element?

-Aman

P.S. What are the allowed ratios? Is there a list of them?

**Comment by markdumay** [comment, marked as answer]
You can find the ratios here: https://gethinode.com/docs/components/image/#arguments

If you do not specify the ratio, the original aspect ratio should be maintained. You might want to adjust the corner ratio though (by default, images are rounded). For example, add the following to `assets/scss/theme/theme.scss` to remove all rounding from regular images:

```scss
img.rounded {
    --bs-border-radius: 0;
}
```

If the original aspect ratio is not preserved, would you mind sharing a (minimal) example so I can reproduce the issue and investigate it?

**Reply by Aman-Aalam** [reply]
Thanks!
The mistake I was making - I specified `ratio=""` but I think I was supposed to omit it all together, which worked.

Thanks again!
